kolibri/utils/options.py

```diff
@@ -304,135 +257,207 @@ def origin_or_port(value):
                 "noeviction",
             ),
             "default": "",
-            "envvars": ("KOLIBRI_CACHE_REDIS_MAXMEMORY_POLICY",),
+            "description": "Eviction policy to use when using Redis for caching, Redis only.",
         },
     },
     "Database": {
         "DATABASE_ENGINE": {
             "type": "option",
             "options": ("sqlite", "postgres"),
             "default": "sqlite",
-            "envvars": ("KOLIBRI_DATABASE_ENGINE",),
+            "description": "Which database backend to use, choices are sqlite or postgresql",
+        },
+        "DATABASE_NAME": {
+            "type": "string",
+            "description": """
+                For SQLite - the name of a database file to use for the main Kolibri database.
+                For Postgresql, the name of the database to use for all Kolibri data.
+            """,
         },
-        "DATABASE_NAME": {"type": "string", "envvars": ("KOLIBRI_DATABASE_NAME",)},
         "DATABASE_PASSWORD": {
             "type": "string",
-            "envvars": ("KOLIBRI_DATABASE_PASSWORD",),
+            "description": "The password to authenticate with when connecting to the database, Postgresql only.",
+        },
+        "DATABASE_USER": {
+            "type": "string",
+            "description": "The user to authenticate with when connecting to the database, Postgresql only.",
+        },
+        "DATABASE_HOST": {
+            "type": "string",
+            "description": "The host on which to connect to the database, Postgresql only.",
+        },
+        "DATABASE_PORT": {
+            "type": "string",
+            "description": "The port on which to connect to the database, Postgresql only.",
         },
-        "DATABASE_USER": {"type": "string", "envvars": ("KOLIBRI_DATABASE_USER",)},
-        "DATABASE_HOST": {"type": "string", "envvars": ("KOLIBRI_DATABASE_HOST",)},
-        "DATABASE_PORT": {"type": "string", "envvars": ("KOLIBRI_DATABASE_PORT",)},
     },
     "Server": {
         "CHERRYPY_START": {
             "type": "boolean",
             "default": True,
-            "envvars": ("KOLIBRI_CHERRYPY_START",),
+            "description": "DEPRECATED - do not use this option, use the 'kolibri services' command instead.",
```

Comment:
We maintain a deprecated option to warn those that may still be using it?

Reply:
Yes - I can add some logic here to give a more explicit warning if this option is still used.

Reply:
If it's a warning, _more explicit_ is always good! 😉

Reply:
Hrm, can't find a good way to do this, so going to skip for now.